crates/sui-adapter-transactional-tests/tests/children/child_of_shared_object.exp

```diff
@@ -24,7 +24,7 @@ task 6 'run'. lines 86-88:
 written: object(109), object(111), object(112)
 
 task 7 'run'. lines 89-89:
-Error: When an (either direct or indirect) child object of a shared object is passed as a Move argument,either the child object's type or the shared object's type must be defined in the same module as the called function. This is violated by object fake(109) (defined in module 'T1::O1'), whose ancestor fake(111) is a shared object (defined in module 'T2::O2'), and neither are defined in this module 'T1::O1'
+Error: MiscellaneousError
```

Comment:
This kind of sucks for these tests. Is there anything we can do for displaying/getting more info out of the adapter/execution_engine for these?

Reply:
Hopefully this is addressed now.